tools/setup-windows/setup-windows.csproj

```
@@ -1,38 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{73DF9E10-E933-4222-B8E1-F4536FFF9FAD}</ProjectGuid>
+    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <OutputType>Exe</OutputType>
     <RootNamespace>setupwindows</RootNamespace>
     <AssemblyName>setup-windows</AssemblyName>
+    <OutputPath>$(XamarinAndroidSourcePath)bin\$(Configuration)-$(TargetFramework)\</OutputPath>
```

If we are `net472` only we could just do:
```suggestion
    <OutputPath>$(XamarinAndroidSourcePath)bin\$(Configuration)\bin\</OutputPath>
```